apps/console/src/features/connections/components/edit/forms/components/common-pluggable-component-form.tsx

```diff
@@ -60,6 +60,19 @@ export const CommonPluggableComponentForm: FunctionComponent<CommonPluggableComp
     // Used for field elements which needs to listen for any onChange events in the form.
     const [ dynamicValues, setDynamicValues ] = useState<CommonPluggableComponentInterface>(undefined);
     const [ customProperties, setCustomProperties ] = useState<string>(undefined);
+    const [ privateKeyValue, setPrivateKeyValue ] = useState<string>(undefined);
+
+    useEffect(() => {
+        dynamicValues?.properties?.map(
+            (prop: CommonPluggableComponentPropertyInterface) =>
+            {
+                if (prop.key === "google_prov_private_key") {
```

Comment:
Shall we move "google_prov_private_key" to a constant [1]?

[1] https://github.com/wso2/identity-apps/blob/master/apps/console/src/features/connections/constants/connection-constants.ts